Userland/Libraries/LibJS/Tests/test-common.js

```diff
@@ -611,4 +612,26 @@ class ExpectationError extends Error {
             duration: 0,
         };
     };
+
+    withinSameSecond = callback => {
+        let tries = 0;
+        let callBackTime = 0;
+        while (tries++ < 5) {
```

Comment:
commit message claims we do 25 retries

Reply:
also, pls use a for loop for readability

Reply:
Yeah reduced the amount since 25 seemed a bit much